src/meta-srv/src/service/cluster.rs

```diff
@@ -31,6 +32,8 @@ impl cluster_server::Cluster for MetaSrv {
                 header: Some(is_not_leader),
                 ..Default::default()
             };
+
+            warn!("The current meta is not leader, but a batch_get request have reached the meta.");
```

Comment:
Better to provide more detailed information like some context or request body if you decide to log them. Same to line 60.